archive/imo/imo1964_q1.lean

```diff
@@ -0,0 +1,103 @@
+/-
+Copyright (c) 2020 Kevin Buzzard. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Kevin Buzzard
+-/
+
+import tactic
+import data.nat.modeq
+
+/-!
+# IMO 1964 Q1
+
+(a) Find all positive integers $n$ for which $2^n-1$ is divisible by $7$.
+
+(b) Prove that there is no positive integer $n$ for which $2^n+1$ is divisible by $7$.
+
+We define part (a) as a predicate, and prove that it is the set of positive
+integers which are a multiple of 3.
+-/
+
+/-!
+## intermediate lemmas
+-/
+
+open nat.modeq
+
+lemma two_pow_mul_three_mod_seven (m : ℕ) : 2 ^ (3 * m) ≡ 1 [MOD 7] :=
```

Comment:
Shouldn't this be `two_pow_three_mul_...`.
Idem for the lemmas below.

Reply:
Thanks. Changed.